src/action.ts

```diff
@@ -12,6 +12,13 @@ import { getDefectDojoInputs, retrieveDefectDojoResults } from "./defect-dojo";
 import { getGitHubContext, getTempDir } from "./github";
 import { getContrastInputs, retrieveContrastResults } from "./contrast";
 
+interface SonarResults {
+  totalResults: number;
+  results: any
+}
+
+const MAX_PAGE_SIZE = 500;
```

Comment:
@drdavella if you would like to test it, you will need to update this value and upload the dist/index.js file